### PR TITLE
Apply proper motion on aladin target overlay

### PR DESCRIPTION
**Pull request by cquiroz**
https://user-images.githubusercontent.com/3615303/168156538-3e16d08c-2ed8-4712-80f9-0d8ab7be60f7.mp4

### CODE REVIEW

File: explore/src/main/scala/explore/targeteditor/AladinContainer.scala [hunk 1]
```diff
@@ -223,26 +226,48 @@ object AladinContainer {
...
+                  target = baseCoordinatesForAladin,
+                  fov = props.options.fovAngle,
+                  showGotoControl = false,
+                  customize = includeSvg _
+                )
+              }
+            )
+          }
+
+          // )
```
Comment (rpiaggio) [inline explore/src/main/scala/explore/targeteditor/AladinContainer.scala:270]: Commented line

**Review by rpiaggio — APPROVED** [review]
LGTM

**Comment by rpiaggio** [comment]
Looks great!

I wonder if we can put the observation time picker on a new header for the whole observations (above the tiles). Maybe an empty tile will do?

**Comment by swalker2m** [comment]
Very nice!  Will the time picker have a "Now" option perhaps?  And operators will want a "Now + Setup" eventually.

**Comment by cquiroz** [comment]
> Looks great!
> 
> I wonder if we can put the observation time picker on a new header for the whole observations (above the tiles). Maybe an empty tile will do?

That was an option and it make sense as to apply to every part of the observation

**Comment by cquiroz** [comment]
> Very nice! Will the time picker have a "Now" option perhaps? And operators will want a "Now + Setup" eventually.

I certainly miss a `Now` option. what `Now + Setup` means?

**Pull request merged**